src/posthog-core.ts

```diff
@@ -864,6 +864,28 @@ export class PostHog {
         return data
     }
 
+    /**
+     * Capture a page view event.
+     *
+     * This is a wrapper around capture and is
+     * equivalent to capture('$pageview', {'$current_url': <url>})
+     *
+     * ### Usage:
+     *
+     *    posthog.page('https://example.com')
+     *
+     * @param {String} url The URL of the page. This will be included in the properties as '$current_url'.
+     * @param {Object} [properties] A set of properties to include with the event you're sending. These describe the user who did the event or details about the event itself.
+     * @param {Object} [config] Optional configuration for this capture request.
+     * @param {String} [config.transport] Transport method for network request ('XHR' or 'sendBeacon').
+     * @param {Date} [config.timestamp] Timestamp is a Date object. If not set, it'll automatically be set to the current time.
```

Comment:
those params dont exist here in the method signature

Reply:
@marandaneto I would like your help with a point that I am confused by.

If you observe the docs for the capture method in lines 761-763 in the same file, it defines some config params, while the param is named options in the function argument.

For the capture method:
1. Is config the same as options?
2. Should config be renamed to options? Shall I do it in this PR itself?

I am trying to reuse the docs of capture method for the new page method.

For the new page method:
1. Should I similarly rename config to be options in the docs?
(or)
2. Should I remove the config lines in the docs for this new method altogether?

Reply:
I think the code docs are not up to date.
`config` is the equivalent of `options`.
timestamp and transport are part of `options`, so they could be removed I guess.
Maybe @pauldambra knows more about this, maybe we just need to update the code docs.

Reply:
Yep, it'll be comments getting out of date.

You're welcome to update the comment on the main capture block has well.